### ISSUE TITLE
Unable to login to audible,  application closes 

### ISSUE THREAD

**Issue by ejheng123** [openaudible/openaudible#1468]
Ubuntu 24.04 lts  open audible 4.4.7   application closes when trying to login to audible.

**Comment by openaudible** [comment]
Try: 
sudo apt install libwebkit2gtk-4.1-0

**Comment by ejheng123** [comment]
I entered the first line at a terminal as recommended and the results
followed.  

  

sudo apt install libwebkit2gtk-4.1-0  

  

libwebkit2gtk-4.1-0 is already the newest version (2.44.3-0ubuntu0.24.04.1).  

libwebkit2gtk-4.1-0 set to manually installed  

  

0 upgraded, 0 newly installed, 0 to remove and 22 not upgraded.  

  

  

I restarted OpenAudible tried to connnect to Audible. It didn't solve the
problem, same result.  

It reaches the login process, I input an email address and password, it then
prompts for a captcha type entry and then abends and closes. Similar result
when entering phone number and OTP.  

  

current log entry on restarting  

  

Oct 12, 2024 5:38:35 PM org.openaudible.desktop.swt.gui.GUI <init>  

INFO: Starting OpenAudible build 4.4.7 for gtk swt 4963 jvm 18.0.1  

Oct 12, 2024 5:38:35 PM org.openaudible.Directories load  

INFO: Library path set to /home/e/OpenAudible/books  

Oct 12, 2024 5:38:35 PM org.openaudible.desktop.swt.manager.AudibleGUI <init>  

INFO: audible desktop 4.4.7 linux  

Oct 12, 2024 5:38:35 PM org.openaudible.desktop.swt.util.shop.PaintShop
prefer2x  

INFO: useHighDPI=Point {96, 96}, threshold is 140. 2x images=false  

Oct 12, 2024 5:38:36 PM
org.openaudible.desktop.swt.manager.player.MiniPlayerView lambda$new$1  

INFO: time size changed: diff=0 bounds=Rectangle {238, 11, 72, 18}  

Oct 12, 2024 5:38:36 PM org.openaudible.library.BookLibrary load  

INFO: Loading /home/e/OpenAudible/books.json length 2434223  

Oct 12, 2024 5:38:36 PM org.openaudible.library.BookLibrary setNeedSave  

INFO: Change to library, need save flag set  

Oct 12, 2024 5:38:37 PM org.openaudible.desktop.swt.manager.AudibleGUI
applicationStarted  

INFO: applicationStarted took 1206 ms. to complete.  

Oct 12, 2024 5:38:40 PM org.openaudible.desktop.swt.manager.VersionCheck
getVersion  

INFO: versionCheck: <https://openaudible.org/swt_version.json>  

Oct 12, 2024 5:39:00 PM org.openaudible.desktop.swt.manager.menu.CommandCenter
execute  

INFO: Command: Console

**Comment by ejheng123** [comment]
Just for completion - here are screen shots of my account and library
settings.  
[did not come through]

**Comment by openaudible** [comment]
Screenshots didn't come through. If the application crashes, the log window won't have anything. But if you run the debug_linux.sh in the OpenAudible directory from the command  you may get a good error.. We don't get many crashes, so would be interested. 
I have recently tested on Kubuntu 24.04, and Ubuntu Desktop 24.04.1 LTS (but not with OTP)

Let me know the output of running from the command line the debug script after it crashes.. you can also send to us via our contact form.

**Comment by ejheng123** [comment]
Thank you for the follow up.  

  

The problem is resolved - but I can't show why.  

  

I have a second computer with the same set up. I was able to login in
successfully. I wondered if it might be a difference because of a VPN
connection. I turned off the VPN on the original machine and was successful. I
turned the VPN back on and still had success.  

  

So the issue is closed as we used to say 'it was a personal issue' in that it
impacted one person for reasons unclear.
